build-system/tasks/extension-helpers.js

```diff
@@ -507,6 +511,46 @@ async function buildExtension(
   await buildExtensionJs(extDir, name, version, latestVersion, options);
 }
 
+/** @type {TransformCache} */
+let cssCache;
```

Comment:
For self-documentation, perhaps call this `jssCache` so readers of this code can clearly see that it's writing to the `.jss-cache` directory?

Reply:
Agreed. Done